dask_ml/model_selection/_incremental.py

```diff
@@ -329,7 +331,7 @@ def fit(
     ...                            chunks=100000, random_state=0)
 
     >>> from sklearn.linear_model import SGDClassifier
-    >>> model = SGDClassifier(tol=1e-3, penalty='elasticnet', random_state=0)
+    >>> est = SGDClassifier(tol=1e-3, penalty='elasticnet', random_state=0)
```

Comment:
slightly easier to review if these stylistic changes are saved for a different PR.